Frameworks/UIKit/UIScrollView.mm

```diff
@@ -250,6 +291,14 @@ - (void)_initScrollViewer:(WXFrameworkElement*)xamlElement {
     _scrollViewer.maxZoomFactor = 1.0f;
     _scrollViewer.minZoomFactor = 1.0f;
     _scrollViewer.zoomMode = WXCZoomModeDisabled;
+
+    // setting transparent background so we can receive touch input
+    WUColor* convertedColor = ConvertUIColorToWUColor([UIColor clearColor]);
+    WUXMSolidColorBrush* brush = [WUXMSolidColorBrush makeInstanceWithColor:convertedColor];
```

Comment:
at which point you could just do  _contentCanvas.background = [WUXMSolidColorBrush makeInstanceWithColor:[WUColor transparent]] 

---
In reply to: [86439907](https://github.com/Microsoft/WinObjC/pull/1305#discussion_r86439907) [](ancestors = 86439907)